.github/workflows/invalidate-prs.yaml

```diff
@@ -0,0 +1,26 @@
+# This workflow invalidates the builds of pull requests that more than 3 hours old.
+#
+
+name: "Invalidate PRs after 3 hours"
+
+on:
+  workflow_dispatch:
```

Comment:
Can you add an input to the dispatch for testing purposes?

It'd look something like this

```yaml
inputs:
      debug:
        description: 'Enable debug logging'     
        required: false
```

with the script task using it via `${{ github.event.inputs.debug }}` as an environment variable.

I'm looking at [this](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs) page for reference.